<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OracleServerClass</class>
 <widget class="QMainWindow" name="OracleServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OracleServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>502</width>
         <height>48</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Record</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="update">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Update Exists</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Clients:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="clientNumber">
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string notr="true"/>
            </property>
            <property name="prefix">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="startRecord">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopRecord">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop and Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>502</width>
         <height>48</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Replay</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="dontKillMe">
          <property name="text">
           <string>Don't kill Chiwawa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testSpecific">
          <property name="text">
           <string>Specific Test</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="smokeBtn">
          <property name="text">
           <string>Smoke Test</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="regBtn">
          <property name="text">
           <string>Run all test case</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopReplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Record Log</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="logTab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Log</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTreeWidget" name="log">
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Message</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Preference"/>
   </widget>
   <addaction name="menu_Edit"/>
  </widget>
  <action name="action_Preference">
   <property name="text">
    <string>&amp;Preference</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../oracleserver.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>regBtn</sender>
   <signal>clicked()</signal>
   <receiver>OracleServerClass</receiver>
   <slot>slot_startRegression()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smokeBtn</sender>
   <signal>clicked()</signal>
   <receiver>OracleServerClass</receiver>
   <slot>slot_startSmokeTest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_replayMCR()</slot>
  <slot>slot_createMCR()</slot>
  <slot>slot_remoteClient()</slot>
  <slot>slot_startRegression()</slot>
  <slot>slot_startSmokeTest()</slot>
 </slots>
</ui>
